src/control/lib/spdk/nvme.go

```diff
@@ -105,23 +108,60 @@ type DeviceHealth struct {
 	VolatileWarn    bool
 }
 
+func (n *Nvme) cleanLockFiles(log logging.Logger, pciAddrs ...string) {
+	log.Debugf("removing lockfiles: %v", pciAddrs)
+
+	for _, pciAddr := range pciAddrs {
+		fName := lockfilePathPrefix + pciAddr
+
+		err := os.Remove(fName)
+		if err != nil {
+			if os.IsNotExist(err) {
+				continue
+			}
+			log.Errorf("remove %s: %s", fName, err)
```

Comment:
I'm not sure we should log and continue here. If we can't clean up a lock file, then it will hang around to cause problems down the road when it's less clear what the problem is.